kube/kubetypes/types.go

```diff
@@ -48,6 +48,7 @@ const (
 	PodIPv4Header string = "Pod-IPv4"
 
 	EgessServicesPreshutdownEP = "/internal-egress-services-preshutdown"
+	ServePreshutdownEP         = "/internal-serve-preshutdown"
```

Comment:
For consistency with egress ep name:

```suggestion
/internal-ingress-services-preshutdown
```